recipes/centos_vertx/Dockerfile

```diff
@@ -16,7 +16,12 @@ ARG VERTX_VERSION=3.4.2
 ENV VERTX_GROUPID=io.vertx 
 
 COPY install-vertx-dependencies.sh /tmp/
+COPY ./run.sh ${HOME}/
 RUN sudo chown user:user /tmp/install-vertx-dependencies.sh && \
     chmod +x /tmp/install-vertx-dependencies.sh && \
     scl enable rh-maven33 /tmp/install-vertx-dependencies.sh && \
-    sudo rm -f /tmp/install-vertx-dependencies.sh
+    sudo rm -f /tmp/install-vertx-dependencies.sh && \
+    sudo chown user:user ${HOME}/run.sh && \
+    sudo chmod +x ${HOME}/run.sh
+
+CMD ["sh","-c","${HOME}/run.sh"]
```

Comment:
newline + same comments than before

Reply:
here you want me to avoid new line or add new line after Line 27 ?